client/extensions/woocommerce/state/wc-api/reducer.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import error from './error-reducer';
+import {
+	SERIALIZE,
+	DESERIALIZE,
+} from 'state/action-types';
+
+const initialState = {};
+
+const handlers = {
+	...error,
+};
+
+export default function( state = initialState, action ) {
+	const { type, payload } = action;
+	const { siteId } = payload || {};
+	const handler = handlers[ type ];
+
+	if ( handler ) {
+		// Necessary to ensure this data is not persisted.
+		if ( type === SERIALIZE || type === DESERIALIZE ) {
```

Comment:
It's not looking for only an error handler. It's looking for a handler of any kind. There will be many more handlers like the one I add in #13545. There's only the error handler in this PR to keep the size down.